test/notifier_test.py

```diff
@@ -16,6 +20,7 @@ def test_single_bus(self):
         bus.send(msg)
         self.assertIsNotNone(reader.get_message(1))
         notifier.stop()
+        bus.shutdown()
```

Comment:
Just a note: These could also use the bus as a context manager.

Reply:
True, but maybe we need to make Notifier a context manager too, otherwise one ends up needing to use try: except: anyway.